cla_public/static-src/javascripts/modules/find-legal-adviser.js

```diff
@@ -307,3 +307,18 @@
     }
   };
 
+  function setMapHeight(setHeight) {
+    if ($(window).width() > 640) {
+      if ($("ul.org-list").height() > setHeight) setHeight = $("ul.org-list").height();
```

Comment:
It probably doesn't make much odds in practice in this context but re-evaluating `$("ul.org-list")` is inefficient, I think. Could be improved by setting it as a var or, in this case, going straight to setting the element height as a var.
Probably not be needed performance wise, just thought it worth mentioning as I was here, and also may make the code read easier.

Reply:
Done